presto-docs/src/main/sphinx/connector/memory.rst

```diff
@@ -92,6 +92,15 @@ To delete an existing table:
 
 .. note:: After using ``DROP TABLE``, memory is not released immediately. It is released after the next write access to the memory connector.
 
+ALTER VIEW
+^^^^^^^^^^
+
+Alter view operations to alter the name of an existing view to a new name is supported in the Memory connector::
```

Comment:
```suggestion
Alter view operations to alter the name of an existing view to a new name is supported in the Memory connector.
```
Same - using `::` here causes the code block to look like this in a local doc build: 

<img width="974" alt="Screenshot 2024-11-13 at 10 48 00 AM" src="https://github.com/user-attachments/assets/97b53a84-9706-4e8b-96a6-392bd9d23796">

I tested this edit in a local doc build and it fixes the code block.